lib/Humming-Bird/Core.rakumod

```diff
@@ -408,23 +420,49 @@ sub group(@routes, @middlewares) is export {
     .(@middlewares) for @routes;
 }
 
-sub routes(--> Hash) is export {
+multi sub advice(--> List:D) is export {
+    @ADVICE.clone;
+}
+
+multi sub advice(@advice) is export {
+    @ADVICE.push: |@advice;
```

Comment:
If I remember correctly, this is exactly what `@ADVICE.append: @advice` would do. https://new-raku.finanalyst.org/type/Array.html#method_append